package.json

```diff
@@ -17,19 +17,19 @@
     "setup"
   ],
   "dependencies": {
-    "@actions/core": "^1.2.4",
+    "@actions/core": "^1.10.1",
     "@actions/github": "^3.0.0",
-    "node-fetch": "^2.6.0"
+    "node-fetch": "^2.7.0"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.13",
-    "@types/node": "^12.0.4",
-    "@types/node-fetch": "^2.5.7",
-    "@zeit/ncc": "^0.21.1",
-    "jest": "^24.8.0",
-    "jest-circus": "^24.7.1",
-    "ts-jest": "^24.2.0",
-    "typescript": "^3.5.1"
+    "@types/jest": "^29.5.8",
+    "@types/node": "^20.12.7",
+    "@types/node-fetch": "^2.6.9",
+    "@vercel/ncc": "^0.38.1",
+    "jest": "^29.7.0",
+    "jest-circus": "^24.9.0",
```

Comment:
Nit: This is the default test runner as of Jest v26, so we should be able to remove this dependency, and remove this entry from the Jest config, without it resulting in any changes to the tests.